src/Microsoft.ML.Onnx/OnnxContextImpl.cs

```diff
@@ -246,10 +248,91 @@ public void AddInputVariable(ColumnType type, string colName)
             _inputs.Add(OnnxUtils.GetModelArgs(type, colName));
         }
 
+        /// <summary>
+        /// Adds constant tensors into the graph.
+        /// </summary>
+        public override string AddInitializer(float value, string name = null)
+        {
+            if (name != null)
+                name = AddVariable(name);
+            else
+                name = AddVariable("initializer");
```

Comment:
Would it make sense to have the default case be a little more descriptive than "initializer"? Since someone might want to look at the generated onnx code (if it is changed to json format), and it would be nice to have it be at least "float initializer", also to reduce clashes in naming between nodes.

Reply:
Ok. Defaults are changed to float, int64, string, and so on. Btw, variable names need to follow C-style naming, so space between words is now allowed. :)